sycl/include/sycl/handler.hpp

```diff
@@ -3207,6 +3351,9 @@ class __SYCL_EXPORT handler {
 
   // Set value of the gpu cache configuration for the kernel.
   void setKernelCacheConfig(sycl::detail::pi::PiKernelCacheConfig);
+
+  sycl::detail::pi::PiImageCopyFlags
+  getPiImageCopyFlags(sycl::usm::alloc SrcPtrType, sycl::usm::alloc DstPtrType);
```

Comment:
I just realized we export this. Since it is only used inside the handler.cpp file I don't think we need it to be a member.

Reply:
Noted, I've moved this function into the `.cpp` file.